sdk/src/utils.ts

```diff
@@ -299,3 +299,11 @@ const isArray = (value: any): boolean => {
 const isObject = (value: any): boolean => {
   return typeof value === 'object' && value !== null
 }
+
+export const sqrtPriceToPrice = (sqrtPrice: SqrtPrice): bigint => {
```

Comment:
Add the type `Price`. This type will help to avoid mistakes when passing `Price` and `SqrtPrice`